fastlane/lib/fastlane/fastlane_require.rb

```diff
@@ -62,6 +62,13 @@ def find_gem_name(user_supplied_name)
 
         return gems.first
       end
+
+      def format_gem_require_name(gem_name)
+        # from "fastlane-plugin-xcversion" to "fastlane/plugin/xcversion"
+        gem_name = gem_name.tr("-", "/") if gem_name.start_with? "fastlane-plugin-"
```

Comment:
Ahhh, good call good call - that is probably a lot safer. @ohwutup, i think you mentioned you did work on the initial plugin stuff. Is this a better way of doing it? ^